src/GNUmakefile

```diff
@@ -10,16 +10,12 @@ else
 	LIB_PATH = /lib
 endif
 
-ifeq ($(shell cat /etc/issue | cut -d " " -f 1), Debian)
```

Comment:
Well, yes, I like the idea to provide some universal mechanism to set PATH for all distributions, but right now this fix brokes the install make target for openSUSE TW x86_64:

```
# make install
install -m 755 pam2control.so /usr/lib/i386-linux-gnu/security/
install: target '/usr/lib/i386-linux-gnu/security/' is not a directory: No such file or directory
make: *** [GNUmakefile:27: install] Error 1
```